MINIFICPP-2239 Add bandwidth limit to InvokeHTTP processor (#1674)
apache/nifi-minifi-cpp: PROCESSORS.md

| Penalize on "No Retry"                          | false                    | true<br/>false              | Enabling this property will penalize FlowFiles that are routed to the "No Retry" relationship.                                                                                                                                                                                                                  |
| **Invalid HTTP Header Field Handling Strategy** | transform                | fail<br/>transform<br/>drop | Indicates what should happen when an attribute's name is not a valid HTTP header field name. Options: transform - invalid characters are replaced, fail - flow file is transferred to failure, drop - drops invalid attributes from HTTP message                                                                |

> | **Upload Speed Limit**                          | 0 B/s                    |                             | Maximum data per second to send, use 0 B/s for unlimited.                                                                                                                                                                                                                                                       |
> | **Download Speed Limit**                        | 0 B/s                    |                             | Maximum data per second to receive, use 0 B/s for unlimited.                                                                                                                                                                                                                                                    |

Reviewer: It would be better to treat an empty value as no limit. We can also keep treating "0 B/s" as no limit, but I'd document the empty as unlimited instead.
```suggestion
| **Upload Speed Limit**                          | 0 B/s                    |                             | Maximum data per second to send. Leave this empty if you want no limit.                                                                                                                                                                                                                                                       |
| **Download Speed Limit**                        | 0 B/s                    |                             | Maximum data per second to receive. Leave this empty if you want no limit.                                                                                                                                                                                                                                                    |
```

> | Download Speed Limit                            |                          |                             | Maximum data per second to receive (e.g. '500 KB/s'). Leave this empty if you want no limit.                                                                                                                                                                                                                    |

Author: Updated in d305cce439c0b60fac04b646b79f46f3a4e68d54
